packages/ng/date/select/translations.ts

```diff
@@ -0,0 +1,65 @@
+﻿export const Translations = {
```

Comment:
These are the same as the date input translations. Maybe we could mutualize them later :)

Reply:
A lot of translations are duplicated we could improve that!

Reply:
Yeah, I noticed that further in the review. Let's do that later©